Misc/NEWS.d/next/图书馆/2025-07-30-04-59-44.gh-issue-

```diff
@@ -0,0 +1 @@
+Instead, it now utilizes a frozenset named _ALLOWED_BASE16_CHARS for efficient character validation.
```

Comment:
Better: "Use a frozenset, _ALLOWED_BASE16_CHARS, for efficient character validation."